pkg/config/configuration.go

```diff
@@ -543,8 +543,14 @@ func IsOperationAllowedByAccessControlPolicy(spiffeID *SpiffeID, srcAppID string
 	if found {
 		// The ACL might have the operation specified as /invoke/*. Here "/*" is stored as the postfix.
 		// Match postfix
-		if operationPolicy.OperationPostFix != "/*" && !strings.HasPrefix(operationPolicy.OperationPostFix, inputOperationPostfix) {
-			return isActionAllowed(action), actionPolicy
+		if strings.Contains(operationPolicy.OperationPostFix, "/*") {
+			if !strings.HasPrefix(inputOperationPostfix, strings.ReplaceAll(operationPolicy.OperationPostFix, "/*", "", -1)) {
+				return false, actionPolicy
+			}
+		} else {
+			if operationPolicy.OperationPostFix != inputOperationPostfix {
+				return false, actionPolicy
```

Comment:
this should be treated as rules do not match and return the upper layer of action i.e. false => isActionAllowed(action)

Reply:
thanks for fixing this :)

Reply:
ok, changed